mod/ckeditor/views/default/elgg/ckeditor.js

```diff
@@ -13,13 +13,19 @@
 define(function (require) {
 	var elgg = require('elgg');
 	require('elgg/init');
+
 	var $ = require('jquery');
 	require('jquery.ckeditor');
+
 	var CKEDITOR = require('ckeditor');
-	var config = require('elgg/ckeditor/config');
 
 	var elggCKEditor = {
-		bind: function (selector) {
+		bind: function (selector, config) {
+			if (config === undefined) {
+				config = require('elgg/ckeditor/config');
```

Comment:
This works, but using sync `require` conditionally gives the wrong impression, so I'd rather set it as `default_config` at the top.